src/Azure.Functions.Cli/Actions/LocalActions/CreateFunctionAction.cs

```diff
@@ -52,6 +58,11 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                 .WithDescription("Function name")
                 .Callback(n => FunctionName = n);
 
+            Parser
+                .Setup<HttpAuthorizationLevel?>('a', "authorizationLevel")
```

Comment:
`authlevel` ? as that matches function.json schema

Reply:
sure i can do that.